README.md

```diff
@@ -1,76 +1,84 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# WEBFLIX (JavaScript Capstone Project)
 
-> One paragraph statement about the project.
+> The JavaScript capstone project is about building our own web application based on an external API(TVmaze API: data about TV series and movies). TVmaze API was selected by the team, because it provides data about a topic that we both like, to enable us build the webapp around it. The webapp will have 2 user interfaces.
+- A home page showing a list of items that you can "like."
+- A popup window with more data about an item that you can use to comment on it
 
-![screenshot](./app_screenshot.png)
+> In this activity, we worked as a team to prepare a Kanban board with a GitHub project. As this our very first group project in our Microverse journey, a very detailed template was provided by Microverse, using a special script prepared for the team.
 
-Additional description about the project and its features.
+![screenshot](./app_screenshot.png)  ![screenshot](./app_popscreenshot.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- JavaScript
+- HTML & CSS
+- Bootstrap
+- Webpack
+- Jest testing framework
+- Git and Github pages
+- VSCode
+- Linter Checkers
 
-## Live Demo
+## Deployment Live Demo
```

Comment:
- in the live version of your app so that kindly update it so that a comment is displayed immediately it is created. This is totally ok in the local server. But in the live demo, I have to close the popup window then reopen it inorder for the comment I had created to appear.

Reply:
- Noted!

Reply:
Done, the live version is working properly now.